tests/search/metrics/metrics.rb

```diff
@@ -24,7 +24,7 @@ def test_metrics
     assert_hitcount("f1:xyzzy", 0)
 
     metrics = vespa.search["test"].first.get_total_metrics
-    dump_metric_names(metrics)
+    # dump_metric_names(metrics)
```

Comment:
Was this change intentional?

Reply:
Yes, I did not see its value. It just polluted the test output.